submissions/dikun/frogger-game/app.js

```diff
@@ -0,0 +1,123 @@
+"use srtict";
+
+const cell = {
+	width: 100,
+	height: 85
+};
+
+const gameField = {
+	top: 0,
+	right: cell.width * 5,
+	bottom: cell.height * 5,
+	left: 0
+};
+
+const playerStart = {
+	x: 202,
+	y: 400
+};
+
+const enemySpeed = {
+	min: 90,
+	max: 250 
+}; 
+
+const sprite = {
+	enemy: 'images/enemy-bug.png',
+	player: 'images/char-boy.png'
+};
+
+const speed = () => enemySpeed.min + Math.floor(Math.random() * enemySpeed.max);
+const crash = 70;
```

Comment:
As a peer developer I want to understand what does `crash` mean and how do I use it. Is this power of crash, overlap by one of axis, overlap amount or anything else?